### ISSUE TITLE
D2x-rebirth with version 575 hosting game and version 709 guests

### ISSUE THREAD

**Issue by D2Lovin** [dxx-rebirth/dxx-rebirth#509]
Just an FYI
If the D2 online game is hosted by version 575, version 709 guests see the reactor immediately explode on game start.  The 575 host and older version guests then get a 800 second countdown timer for reactor life.  If a version 709 hosts the game, all guests function normally.

**Comment by vLKp** [comment]
Please explain in greater detail, and preferably fill out the issue reporting template.  D2X-Rebirth has never had a version 575 or version 709.  All released versions have been 0.x.y, for x < 100 and y < 10.  Additionally, the game is supposed to prevent clients of different versions from joining at all, because the in-game message protocol was never standardized.

Where did you get the source that you built into the version 575?  Likewise for version 709.

**Comment by D2Lovin** [comment]
The version #s are what are listed on the bottom of the main screen in red as D2X-Rebirth v0.61 0.60.0-Beta2-NNN-GFxxxxx* where NNN = the 575 or 709 numbers that I listed.
  All versions were obtained from https://www.dxx-rebirth.com/download-dxx-rebirth/
  There has been a new file every month since Dec. 2019 and the difference is in the NNN version number. I forget the month of the 575 version but the 709 version was the 5-1-2020 update.
   We have not had a problem with players having different NNN version numbers playing in the same game until the latest version.  It's not a big deal as I will get everyone to update to the 5-1-2020 update.  I just wanted to report the issue.

**Comment by vLKp** [comment]
I see.  That is not a version number.  That is a count of commits since the last tag.  It will reset when a new tag is created.  Knowing the commit count, I can work out what commits you used.  Having the xxxx part would be even better though.  If I computed this correctly, the 709 count is commit f0a47c358ba6f084c07562b946ff2f98aa363774 and the 575 count is commit 2bedba0a2c187b11205b51462654ad804ab56f7e, which would make 575 probably the November build, since that commit is dated Oct 28.  Regardless of when it was built, knowing the commits is what I need.  That tells me exactly what code went into each build, and from that what code to test to try to reproduce the problem.

I just went through the changes to the multiplayer code between those two commits and I don't see anything that would account for this.  Does the problem happen on all multiplayer game modes or only certain types?
